presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh

```diff
@@ -6,6 +6,17 @@ CONFIG="$1"
 
 shift 1
 
+# creating a copy of the file_auth file for the PrestoCliFileAuthTests.testAuthTokenCacheUpdate()
+# to avoid duplicate entries in the original file
+if [[ "$CONFIG" == "fileauth" ]]; then
+  # check if the same copy exists -> delete in case it exists
+    fileAuthCopy=/docker/volumes/conf/presto/etc/file_auth_copy
+    if [ -f "$fileAuthCopy" ]; then
+      rm -f $fileAuthCopy
+    fi
+    cp /docker/volumes/conf/presto/etc/file_auth $fileAuthCopy
```

Comment:
I think it would be easier to use volume in `presto-product-tests/conf/docker/fileauth/docker-compose.yml`

Reply:
can we have volume with copy-on-write semantics?
(the file is being modified in the `PrestoCliFileAuthTests.testAuthTokenCacheUpdate` test)

btw, do we need `PrestoCliFileAuthTests.testAuthTokenCacheUpdate()` as a product test? i think unit test would be sufficient for this

Reply:
@electrum Shall we get rid of this test? This will also resolve the "file_auth_copy" issues.